plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -0,0 +1,233 @@
+import {SET, ADD_ITEM, REMOVE_ITEM} from '../../../../../../src/js/constants/TransactionTypes';
+import Transaction from '../../../../../../src/js/structs/Transaction';
+import {combineReducers, simpleFloatReducer} from '../../../../../../src/js/utils/ReducerUtil';
+
+const containerReducer = combineReducers({
+  cpus: simpleFloatReducer('resources.cpus'),
+  mem: simpleFloatReducer('resources.mem'),
+  disk: simpleFloatReducer('resources.disk')
+});
+
+function containersParser(state) {
+  if (state == null || state.containers == null) {
+    return [];
+  }
+  return state.containers.reduce((memo, item, index) => {
```

Comment:
So there should be a return before each `return` ? I hear this rule sometimes and then again I hear it is not like that. Could we find a simple general rule for this? 

Because I feel like: https://www.youtube.com/watch?v=KgEytquYusM